app/src/main/java/com/github/bsideup/liiklus/Application.java

```diff
@@ -58,8 +58,8 @@ public static SpringApplication createSpringApplication(String[] args) {
         environment.setDefaultProfiles("exporter", "gateway");
         environment.getPropertySources().addFirst(new SimpleCommandLinePropertySource(args));
 
-        var pluginsDir = environment.getProperty("plugins.dir", String.class, "./plugins");
-        var pathMatcher = environment.getProperty("plugins.pathMatcher", String.class, "*.jar");
+        var pluginsDir = environment.getRequiredProperty("plugins.dir", String.class);
```

Comment:
why without default now?

Reply:
ah, moved to config

Reply:
yeah, wanted to not have the defaults in code